cypress/integration/accessibility_How To Hack Around Gestalt_spec.js

```diff
@@ -0,0 +1,20 @@
+describe('How to Hack Around Gestalt Accessibility check', () => {
+  beforeEach(() => {
+    cy.visit('/how_to_hack_around_gestalt');
+    cy.injectAxe();
+  });
+
+  // Disable the test for now since it's timing out on GitHub CI
```

Comment:
@dangerismycat what caused the timeout in GitHub CI? Could we file a bug for this issue?

Reply:
no issue, it was an oversight on my part
https://github.com/pinterest/gestalt/pull/1861